docs/concepts/model_endpoints.md

```diff
@@ -83,6 +88,31 @@ endpoint = client.create_model_endpoint(
 )
 ```
 
+## Creating Streaming Model Endpoints
+
+Streaming model endpoints are variants of sync model endpoints that are useful for tasks with strict requirements on perceived latency. Streaming endpoints are more expensive than async endpoints.
+!!! Note
+    Streaming model endpoints require at least 1 `min_worker`.
+
+```py title="Creating a Streaming Model Endpoint"
+import os
+from launch import LaunchClient
+
+client = LaunchClient(api_key=os.getenv("LAUNCH_API_KEY"))
+endpoint = client.create_model_endpoint(
+    endpoint_name="demo-endpoint-streaming",
+    model_bundle="test-bundle",
```

Comment:
Let's use `test-streaming-bundle` here. The idea is that we run integration tests on these docs and we want to ensure that the bundle for this endpoint is a streaming one.